Arkadia/sheet.json

```diff
@@ -0,0 +1,8 @@
+{
+	"html": "arkadia.html",
+	"css": "arkadia.css",
+	"authors": "Zholtar",
+	"roll20userid": "-",

```

Comment:
Please provide your roll 20 user id.  This can be found by clicking into your profile and inspecting the contents of the address bar.  It is the number that comes before your user name.